test/container_test.exs

```diff
@@ -3,15 +3,19 @@ defmodule RemoteDockers.ContainerTest do
   alias RemoteDockers.{Container, HostConfig}
   doctest RemoteDockers.Container
 
+  @host_config HostConfig.build(
+    "https://192.168.99.100",
+    "/path/to/cert.pem",
+    "/path/to/key.pem"
+  )
```

Comment:
unit tests can never works with that.
We need to load configuration from config/test.exs